release-notes/major11/11.1.0/valtimo-frontend-libraries.md

```diff
@@ -42,6 +42,10 @@ The following bugs were fixed:
 
   When using the **valtimo-form-io** component from *@valtimo/components*, the local storage property *formioToken* gets updated every time the token refreshes after expiring.
 
+* **Fixed infinitely loading task list**
```

Comment:
I remember discussing this in the past. This is the bugfixes section, so whatever we list here will be fixed. As such, the idea is to have a title that lets the reader know what the bug is, and if they want to know more they will read the description, where a more proper explanation of the issue is given. So no need to say a bug was fixed. If it wasn't fixed, it wouldn't be here in the list.